sdk/src/snark.rs

```diff
@@ -87,7 +87,7 @@ impl TryFrom<bonsai_sdk::responses::Groth16Seal> for Seal {
 
 /// Snark proof as generated by Bonsai.
 #[derive(Clone, Debug)]
-pub struct Proof {
+pub(crate) struct Proof {
```

Comment:
Same suggestion as for `generate_proof`. Wouldn't this be useful in the context of the ethereum-sdk?